FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -290,6 +290,9 @@ - (void)refreshInstallationsTokenWithCompletionHandler:
 
         FIRLogInfo(kFIRLoggerRemoteConfig, @"I-RCN000022", @"Success to get iid : %@.",
                    strongSelfQueue->_settings.configInstallationsIdentifier);
+        return [strongSelfQueue reportCompletionOnHandler:completionHandler
+                                               withStatus:FIRRemoteConfigFetchStatusNoFetchYet
+                                                withError:nil];
```

Comment:
Because are waiting on this async function, we need to report back for all code paths, such as the success path.